.github/workflows/lint.yml

```
@@ -0,0 +1,27 @@
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up build environment
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y build-essential
+
+    - name: Check if project builds successfully!
+      run: make build
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)